src/components/ListItem.js

```diff
@@ -0,0 +1,44 @@
+import { AiFillEdit } from "react-icons/ai";
+
+export default function ListItem({
+  index,
+  name,
+  isComplete,
+  onTogglePresence,
+  onDelete,
+  handleRenamingToDo,
+}) {
+  function handleDeleteClick() {
+    onDelete(index);
+  }
+
+  function handleToggleClick() {
+    onTogglePresence(name, index);
```

Comment:
 You are passing the `name` as an argument to the function `onTogglePresence`, but as I wrote before in the `App.js`, you are not using it, hence you could remove it